src/app/customer/Customer.tsx

```diff
@@ -393,8 +396,9 @@ export function mapToWithCheckoutCustomerProps(
             privacyPolicyUrl,
             requiresMarketingConsent,
             isSignInEmailEnabled,
+            isAccountCreationEnabled,
         },
-    } = config;
+    } = config as StoreConfig & { checkoutSettings: { isAccountCreationEnabled: boolean } };
```

Comment:
Just need to remember to update this once SDK is updated with the new type definition.